### ISSUE TITLE
Add missed loader for the ReLU and ELU activation layers

### ISSUE THREAD

**Issue by zaleslaw** [JetBrains/KotlinDL#78]
Each layer should have an implementation for export/import from/to JSON configuration (Keras compatible format) (see [ModelLoader.kt ](https://github.com/JetBrains/KotlinDL/blob/master/api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/ModelLoader.kt) and [ModelSaver.kt](https://github.com/JetBrains/KotlinDL/blob/master/api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/ModelSaver.kt))

The saving functions for the ReLU  and ELU activation layer are missed

As an integration example, add a convolutional neural network to the examples package (in CNN package for example), it could be based on [improved LeNet model](https://github.com/JetBrains/KotlinDL/blob/master/examples/src/main/kotlin/examples/cnn/mnist/ModernLeNet.kt), train it on "mnist" or "fashion mnist" and add it as an integration test to the "examples" module tests

- [ ] add saving function for ReLU activation layer
- [ ] add saving function for ELU activation layer
- [ ] add an example
- [ ] convert this example to the integration test
- [ ] train network, tune it to achieve accuracy more than >70%

**Comment by avan1235** [comment]
I can take if there are no volunteers for this issue and I was adding the ELU layer as I don't see other work for now for me and I'd like to try to export the trained model in kotlinDL to see how it works :wink:

**Comment by zaleslaw** [comment]
I suppose you are enough mature for second-level tasks, let's leave it for newcomers, it's easy to add, but need to understand some important things.
If it will be not closed till the end of July, I'll notify you or fix it be myself

**Comment by dosier** [comment]
I can volunteer for this issue. 
I already see there is support for loading **ELU** and **ReLU** layers in [ModelLoader.kt](https://github.com/JetBrains/KotlinDL/blob/master/api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/ModelLoader.kt)
So only the saving part needs to be done right?

**Comment by dosier** [comment]
Nevermind my commits, we happened to work on it at the same time :D

**Comment by mkaze** [comment]
@kokorins @dosier Just a tip: it's much better to first leave a comment mentioning that you are working on a PR for an existing issue in the issue page (or create an issue first if it does not already exist and let the maintainers know that you are interested to work on it). With that, you can prevent duplicated effort and you don't end up working on an issue which another person is also working on at the same time. Further, for some of the issues, you may need to confirm and check with maintainers that you want to start working on it; this is so as to prevent working on an issue which is not really an issue or is not a priority at least in a near future. Currently, the maintainer of this project is @zaleslaw.

**Comment by zaleslaw** [comment]
Yeah, as @dosier we have not yet the contribution guidelines, but looks like this is a time to write something about that.
Sorry for the situation, @dosier

**Comment by zaleslaw** [comment]
@kokorins could you please write any comment here, to assign the ticket on you

**Comment by kokorins** [comment]
I've just read a code a bit and decided to contribute. As there were no action until the Pull Request, I havent expected any parallel work.

Can someone share the code style for the project (IntelliJ xml or something of those lines)?

**Comment by zaleslaw** [comment]
It's ok @kokorins, no code style/contribution guideline in the project yet ( we are working on it).

I've committed some days ago` gradle.properties` file with the setting `kotlin.code.style=official`, so it's enough, for now, to apply Ctrl+Alt+L to the file and remove double blank lines
